### ISSUE TITLE
7.3

### ISSUE THREAD

**Issue by faithfulman18** [InExtenso/win32service#34]
On the home page, it states that 7.3 is supported, but when I went to the PECL site, it seems that there is not a 7.3 version out there.  Is that going to be updated soon since 7.3 is supposed to be GA tomorrow?

**Comment by macintoshplus** [comment]
Hi,

Thanks for your ask. The new version 0.3.1 of the Win32Service is in RC1 for PHP 7.3. You can download the pre-built dll here:
https://github.com/win32service/win32service/releases/tag/v0.3.1-RC1
or here: https://ci.appveyor.com/project/macintoshplus/win32service/branch/less_admin/job/ca2oyi3bn0t361pe/artifacts

The version 0.3.1 will be on pecl.php.net website after the PHP 7.3 releases. Please, send me your feedback for the version.

**Comment by faithfulman18** [comment]
Alright , that sounds good.  I'll be waiting until the official release then.  Thanks again.

**Comment by macintoshplus** [comment]
Hi,

You can wait until the stable version out. But, if I do not have feedback, the final version is this version.

Can you run your PHP services in staging environment with the new version of PHP and Win32Service to validate the change ?

Thanks for your interest and time.

**Comment by macintoshplus** [comment]
The 0.3.1 is available on PECL website here: https://pecl.php.net/package/win32service/0.3.1/windows

**Comment by faithfulman18** [comment]
Hi @macintoshplus,

Sorry I haven't gotten back to you yet.  I have spent a significant amount of time this past weekend working on this.  However, to properly get my project working to test all aspects I ran into a bunch of changes I had to make since it was originally in php 5.5.  While I'm about 80% done with retrofitting my code, I'm not quite there yet. I wasn't ignoring you, I just wanted to give you a sufficient response, but without yet being able to do so.

I cannot tell you how thankful I am for you working on this project to get it available for newer versions of php.  I have made a significant time investment using this plugin and had wondered if I was going to have to rebuild things using another method.

Anyway, I'll try to still provide you feedback once I'm completely done testing.  For now, so far things that I have tested have been working properly.

However, Im not quite sure if I understand the permissions change so that the service can be start/stopped by a non admin account.

Thanks again for your work on this project.

Kevin
